app/controllers/application_controller.rb

```diff
@@ -14,9 +21,11 @@ def switch_locale(&action)
     I18n.with_locale(locale, &action)
   end
 
-  rescue_from ActionPolicy::Unauthorized do |ex|
-    flash[:alert] = t("errors.authorization_error")
-
-    redirect_back fallback_location: root_path
+  def log_and_reraise(error)
+    unless KNOWN_ERRORS.include?(error.class)
+      Bugsnag.notify(error)
+      error.instance_eval { def skip_bugsnag = true }
+    end
+    raise
```

Comment:
Default behavior of BugSnag is to log unhandled errors. This config will also log handled errors, except authz errors. It also should avoid logging unhandled errors twice.

If there are more handled errors we don't want to log, we could add them to the const.